pull_request_template.md

```diff
@@ -1,35 +1,35 @@
 <!-- PR title format: `<Feat|Bug|Test|Doc|Chore|Junk> - <tracker-number> - <Short description>` -->
 <!-- Fill out this PR template to make it easier for reviewers to understand your code. Remove this comment and any unnecessary section. -->
 
-#### What does this PR do?
+# What does this PR do?
```

Comment:
Changed using H1 instead of H4 for the PR section titles, this way we can more easily add subsections with H2 (##), H3 (###), etc